### PR TITLE
workloads: add centos 7

### PR DESCRIPTION
**Pull request by devimc**
Add centos 7 workload that is similar to fedora

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: workloads/centos7.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+base_image_url: https://cloud.centos.org/centos/7/images/CentOS-7-x86_64-GenericCloud-1804_02.qcow2
+base_image_name: Centos 7
+hostname: singlevm
```
Comment (markdryan) [inline workloads/centos7.yaml]: This field is no longer supported and should be deleted.

Comment (markdryan) [reply workloads/centos7.yaml]: And by this field I mean 'hostname'

Comment (markdryan) [reply workloads/centos7.yaml]: @devimc If you just want to remove the hostname field I'll merge this patch this evening.  Everything else looks good.

**Comment by devimc** [comment]
@markdryan changes applied, thanks

**Pull request merged**